src/ROUTES.js

```diff
@@ -17,8 +17,10 @@ export default {
     IOU_BILL: 'iou/split',
     SEARCH: 'search',
     SIGNIN: 'signin',
-    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:validateCode',
+    SET_PASSWORD_WITH_CODE: 'setpassword/:validateCode',
```

Comment:
Hm, this is going to conflict with https://github.com/Expensify/Expensify.cash/pull/1857. We need to be careful there

Reply:
I don't feel super strongly about naming conventions here, `VALIDATE_LOGIN_WITH_VALIDATE_CODE` just seemed kind of redundant. Happy to use that though to avoid conflicts.